drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/factory/ChromeDriverFactory.java

```diff
@@ -77,12 +77,13 @@ public void destroyInstance(ChromeDriver instance) {
     @Override
     public ChromeDriver createInstance(WebDriverConfiguration configuration) {
         final ChromeOptions options = getChromeOptions(configuration);
-        ChromeDriverService chromeDriverService = new ChromeDriverService.Builder()
-            .withLogOutput(System.out).build();
 
-        return SecurityActions.newInstance(configuration.getImplementationClass(),
-            new Class<?>[] { ChromeDriverService.class, ChromeOptions.class },
-            new Object[] { chromeDriverService, options }, ChromeDriver.class);
+        try (ChromeDriverService chromeDriverService = new ChromeDriverService.Builder().build()) {
```

Comment:
@mabartos I traced the performance issue here.
This is causing 5+ second delay in initialization.

Reply:
I have reverted that change. I have debugged the code and `ChromeDriverService.close()` is being called by WebDriver internally, so no need to have `try-with-resources` here

Reply:
Interesting stuff; thanks for the clarification.